starter/js/atm.js

```diff
@@ -1,15 +1,42 @@
+console.log("clint");
+$(document).ready(function () {
+  $('#depositChecking').on('click', checkingAccountDepost)
+  $('#withdrawChecking').on('click', checkingAccountWithdrawl)
+  $('#depositSavings').on('click', savingsAccountDeposit)
+  $('#withdrawSavings').on('click', savingsAccountWithdrawl)
+  $('#withdrawChecking').on('click', overdraftProtection)
+});
+
+  $('body').css('background-color','red');
 //Begin with the document ready function
 
     //Checking account deposit function
+      // var checkingInput = $('#amountChecking').val()
 
-      //On click of the depositChecking button
+      // var checkingWithdawBalance = checkingInput -= checkingBalance
 
-        //Get value from the amountChecking input field
 
-        //Take that value and add it to the existing value in the checkingBalance div
+    function checkingAccountDepost() {
+      var checkingBal = $('#checkingBalance').text().replace('$','');
+      var checkingNum = parseInt(checkingBal);
+      var checkingInput = $('#amountChecking').val()
+      var checkingNumTwo = parseInt(checkingInput);
```

Comment:
Instead of creating four different variables, you could create one `checkingBal` and then just reassign it a new value each time you make a change to it.